packages/cdk-graph-plugin-diagram/test/graphviz/dot.test.ts

```diff
@@ -189,4 +189,32 @@ describe("dot", () => {
       );
     });
   });
+
+  it("should set node position", async () => {
+    const outdir = await makeCdkOutdir("node-position");
+
+    const app = new FixtureApp({ outdir });
+    const plugin = new CdkGraphDiagramPlugin({
+      defaults: {
+        nodePositions: new Map([["WebServer", "0,10!"]]),
```

Comment:
Hmm is `Point` just a string? We'd need to make sure we document some examples if we leave it as a string - I'd have expected an object with x/y properties :)